libs/networking/WordPressNetworking/src/main/java/org/wordpress/android/networking/RestClientUtils.java

```diff
@@ -389,4 +424,49 @@ public void post(final String path, JSONObject params, RetryPolicy retryPolicy,
         AuthenticatorRequest authCheck = new AuthenticatorRequest(request, errorListener, mRestClient, mAuthenticator);
         authCheck.send();
     }
+
+    /**
+     * Returns locale parameter used in REST calls which require the response to be localized
+     */
+    public static HashMap<String, String> getRestLocaleParams(Context context) {
+        HashMap<String, String> params = new HashMap<>();
+        //better use getConfiguration as it has the latest locale configuration change.
+        //Otherwise Locale.getDefault().getLanguage() gets
+        //the config upon application launch.
+        String deviceLanguageCode = context != null ? context.getResources().getConfiguration().locale.toString() : Locale.getDefault().getLanguage();
+        if (!TextUtils.isEmpty(deviceLanguageCode)) {
+            params.put("locale", deviceLanguageCode);
+        }
+        return params;
+    }
+
+    /**
+     * Takes a URL with query strings and separates it into the path and a Map of query strings values.
+     * Returns both objects in a List, where list[0] = path and list[1] = Map<String, String>
+     */
+    public static List sanitizeUrl(String unsanitizedPath){
```

Comment:
I would rather to have 2 different methods `getSanitizedPath` and `getSanitizedParameters`.